core/src/main/java/bio/terra/pearl/core/dao/BaseVersionedJdbiDao.java

```diff
@@ -20,6 +22,7 @@ public List<T> findByStableId(String stableId) {
         return findAllByProperty("stable_id", stableId);
     }
 
+    /** gets 1 plus the previous highest version, or 1 if no published version already exists */
```

Comment:
```suggestion
    /** gets 1 plus the previous highest version, or 1 if no version already exists */
```
Or some other minor change so that it's not exactly the same as the docs for `getNextPublishedVersion`.